lib/tutorials/ko_KR/getting-started.md

```diff
@@ -134,73 +170,78 @@ npm install --save good-squeeze
 'use strict';
 
 const Hapi = require('hapi');
-const Good = require('good');
 
-const server = new Hapi.Server();
-server.connection({ port: 3000, host: 'localhost' });
+const server = Hapi.server({
+    port: 3000,
+    host: 'localhost'
+});
 
 server.route({
     method: 'GET',
     path: '/',
-    handler: function (request, reply) {
-        reply('Hello, world!');
+    handler: (request, h) => {
+
+        return 'Hello, world!';
     }
 });
 
 server.route({
     method: 'GET',
     path: '/{name}',
-    handler: function (request, reply) {
-        reply('Hello, ' + encodeURIComponent(request.params.name) + '!');
-    }
-});
+    handler: (request, h) => {
 
-server.register({
-    register: Good,
-    options: {
-        reporters: {
-            console: [{
-                module: 'good-squeeze',
-                name: 'Squeeze',
-                args: [{
-                    response: '*',
-                    log: '*'
-                }]
-            }, {
-                module: 'good-console'
-            }, 'stdout']
-        }
-    }
-}, (err) => {
+        // request.log(['a', 'name'], "Request name");
+        // or
+        request.logger.info('In handler %s', request.path);
```

Comment:
If we _do_ want to describe usage of hapi-pino, I would personally suggest that we demonstrate its hapi-compatible logging (using `request.log()`).